### DISCUSSION TITLE
The license does not match the Video DownloadHelper browser version

### DISCUSSION THREAD

**Discussion by mamedov** [aclap-dev/video-downloadhelper#478] [License & Purchase requests]
### What's your issue?

Hello,

I have just bought license using Microsoft Edge (chromium) and paid. But when I clicked on "Install" I have got the message: "The license does not match the Video DownloadHelper browser version". I have tried both Chrome and Edge.

Please advise.

Thank you!

### The 10 first digits (only 10) of your license number or email address

c02803a5cb

### I confirm that I did not not paste my full license number

- [X] I confirm

**Comment by mamedov** [comment]
Now I can see in the settings the following message: "License is for Edge but extension build is for Chrome".
I am on Mac os, and I am using Microsoft Edge (chromium based) and was installed extension from Chrome Web Store which is compatible with Edge.

**Comment by mamedov** [comment]
Now I have installed extension from Microsoft Edge store but it is locked. Could you please help me?

**Comment by jcfrog** [comment]
Hi,
You should have received an email to reset your counter. Maybe in your spam box.
Anyway, I have reset your installation counter, should be OK now.

**Comment by jcfrog** [comment]
I confirm your license is for Edge, you can't use it on Chrome.

About licenses: https://github.com/aclap-dev/video-downloadhelper/wiki#go-premium

**Comment by mamedov** [comment]
Thank you! It works now :)
